### ISSUE TITLE
Can't use cached images with ':' in their name

### ISSUE THREAD

**Issue by ignacysokolowski** [testcontainers/testcontainers-node#177]
Hi, I'm running a local docker registry on localhost:5000. I tag my image with the registry address in front of the image name, for example `localhost:5000/my-image:latest`, and I want test that it was built and tagged correctly before pushing it to the registry. The problem is that testcontainers fails to find it in the cache and tries to pull it:

```
(HTTP code 404) unexpected - manifest for localhost:5000/my-image:latest not found: manifest unknown: manifest unknown
```

I found that when `GenericContainer` starts, it first checks if the image is cached by calling `DockerClient.fetchRepoTags`, where each repo tag is split into image name and tag on the **first** colon:

https://github.com/testcontainers/testcontainers-node/blob/ca3b4866d92b233945b439f424f95a4d148c042c/src/docker-client.ts#L245

In case of `'localhost:5000/my-image:latest'`, `imageName` becomes `'localhost'` and `tag` becomes `'5000/my-image'`.

**Comment by cristianrgreco** [comment]
hi @ignacysokolowski, thanks for finding/raising the issue, it'd be great if you could raise a PR, if not i'll get to this in the next few days

**Comment by cristianrgreco** [comment]
fixed in v6.4.2
